modules/cbProcessFlow/pushAlongFlow.php

```diff
@@ -55,6 +58,9 @@ public function process($context = false) {
 		}
 		$processflow = $pflowid;
 		$pffield = $rs->fields['pffield'];
+		if ($module == '') {
+			return '';
+		}
```

Comment:
move this block right under the assignment to $module:  `$module = getSalesEntityType($recid);`
keeping it together makes the code easier to maintain